appveyor.yml

```diff
@@ -0,0 +1,27 @@
+version: '{build}'
+
+skip_tags: true
+
+install: []
+
+environment:
+  matrix:
+    - VSVERSION: Visual Studio 12
```

Comment:
I'd prefer to stay away from older MSVC compilers.  Their C++11 support is sometimes lacking.